### ISSUE TITLE
Problem importing module imports.py: cannot import name 'input' on Python 3

### ISSUE THREAD

**Issue by derekbekoe** [PyCQA/pylint#1518]
### Steps to reproduce
1. Clean virtualenv
2. `pip install pylint`
3. `pylint`

### Current behavior
```
$ pylint
Problem importing module imports.py: cannot import name 'input'
```

### Expected behavior
You don't see the import error.

### pylint --version output
```
$ pylint --version
pylint 1.7.1
```

**Comment by derekbekoe** [comment]
Okay seems like it was due to the `isort` dependency.
https://github.com/timothycrosley/isort/issues/548

**Comment by timothycrosley** [comment]
@derekbekoe is this resolved with isort 4.2.12 for you?

**Comment by derekbekoe** [comment]
@timothycrosley Yes thanks!
